src/modules/UI/components/ControlPanel/Component/UserList.js

```diff
@@ -47,12 +48,9 @@ export default class UserList extends Component<Props> {
     return this.props.deleteLocalAccount(username)
   }
   handlePressDeleteLocalAccount = (username: string) => () => {
-    return Alert.alert(
-      'Delete Account',
-      "Delete '" +
-        username +
-        "' on this device? This will disable access via PIN. If 2FA is enabled on this account, this device will not be able to login without 2FA reset which takes 7 days",
-      [{ text: 'No', style: 'cancel' }, { text: 'Yes', onPress: () => this.handleDeleteLocalAccount(username)() }]
-    )
+    return Alert.alert(s.strings.delete_account_header, s.strings.delete_username_account, [
```

Comment:
This seems to be missing the string replacement for the username. `delete_username_account` has a %1$s but there is no sprintf in the code to insert the username

Reply:
Updated. Done